packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form-view.tsx

```diff
@@ -0,0 +1,355 @@
+import { h } from "@stencil/core";
+import { createStyleSheet } from "../../../styling/JSS";
+import { intl } from "../../../global/global";
+
+export interface BankingInfoFormViewProps {
+  states: {
+    loading: boolean;
+    disabled: boolean;
+    hideSteps: boolean;
+    hideBanking?: boolean;
+    hidePayPal?: boolean;
+    feeCap?: string;
+    isPartner: boolean;
+    formState: {
+      checked: "toBankAccount" | "toPaypalAccount" | undefined;
+      errors?: any;
+    };
+  };
+  slots: {
+    formInputsSlot?: any;
+    countryInputSlot?: any;
+    paymentMethodSlot?: any;
+  };
```

Comment:
I think we use `any` for all our slot types in mint but it might be work looking to see if we can properly type these?